AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientController.swift

```diff
@@ -171,6 +171,26 @@ extension DefaultAudioClientController: AudioClientController {
         }
     }
 
+    func promoteToPrimaryMeeting(credentials: MeetingSessionCredentials, observer: PrimaryMeetingPromotionObserver) {
+        guard Self.state == .started else {
```

Comment:
is this supposes to be capital `Self`?

Reply:
Probably? just copying from rest of code :p